lib/compiler-llvm/src/translator/intrinsics.rs

```diff
@@ -460,6 +465,145 @@ impl<'ctx> Intrinsics<'ctx> {
                 None,
             ),
 
+            // VM libcalls.
+            table_copy: module.add_function(
+                "wasmer_table_copy",
+                void_ty.fn_type(
+                    &[
+                        ctx_ptr_ty.as_basic_type_enum(),
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        i32_ty_basic,
+                    ],
+                    false,
+                ),
+                None,
+            ),
+            table_init: module.add_function(
+                "wasmer_table_init",
+                void_ty.fn_type(
+                    &[
+                        ctx_ptr_ty.as_basic_type_enum(),
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        i32_ty_basic,
+                    ],
+                    false,
+                ),
+                None,
+            ),
+            table_fill: module.add_function(
+                "wasmer_table_fill",
+                void_ty.fn_type(
+                    &[
+                        ctx_ptr_ty.as_basic_type_enum(),
+                        i32_ty_basic,
+                        i32_ty_basic,
+                        funcref_ty.as_basic_type_enum(),
```

Comment:
here as well, is `funcref_ty` the thing we're using for all reference types? Might be better to give it a different name when it's used in opaque contexts, as we do deref a funcref_ty in call indirect